### PR TITLE
 Fixing retweet bug

### PR DESCRIPTION
**Pull request by erikschlegel**

### CODE REVIEW

**Review by jcjimenez — APPROVED** [review]
LGTM

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/TwitterStreamFactory.scala [hunk 1]
```diff
@@ -62,14 +62,18 @@ class TwitterStreamFactory(configurationManager: ConfigurationManager) extends S
       query = Some(query)
     )
 
+    def isRetweet(status: Status) : Boolean = {
```
Comment (jcjimenez) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/TwitterStreamFactory.scala]: `s/isRetweet/isOriginalTweet/`

File: src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/TwitterStreamFactory.scala [hunk 2]
```diff
@@ -62,14 +62,18 @@ class TwitterStreamFactory(configurationManager: ConfigurationManager) extends S
...
+
     params.getOrElse("trustedSourceFilterEnabled", "true").toString.toBoolean match {
       case false => stream
       case true => {
         val trustedSourceScreenNames = configurationManager.fetchTrustedSources(sparkContext = ssc.sparkContext)
           .filter(source=>source.pipelinekey.equalsIgnoreCase("twitter"))
           .map(source=>source.externalsourceid).toSet
 
-        stream.filter(status=>{ trustedSourceScreenNames.contains(status.getUser.getScreenName) && !status.isRetweet })
+        stream.filter(status=>{ trustedSourceScreenNames.contains(status.getUser.getScreenName) && !isRetweet(status) })
```
Comment (jcjimenez) [inline src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/TwitterStreamFactory.scala]: `s/!isRetweet/isOriginalTweet/`

**Review by jcjimenez — APPROVED** [review]
LGTM

**Review by kevinhartman — APPROVED** [review]
Please squash merge :)

**Pull request merged**